packages/wallet/frontend/src/utils/helpers.ts

```diff
@@ -57,3 +57,15 @@ export const formatDate = (date: string): string => {
     minute: '2-digit'
   })
 }
+
+export const getFeesValue = (
+  send: FormatAmountArgs,
+  receive: FormatAmountArgs
+): FormattedAmount => {
+  const fees = Number(send.value) - Number(receive.value)
```

Comment:
We should not use `Number(/*...*/)`, since everything comes as a `bigint`. In a particular case, if `send.value` is greater than [`Number.MAX_SAFE_INTEGER`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER) we might get some weird results.